### DISCUSSION TITLE
Next.js

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents-react#4516] [Q&A] [answered]
Hello, 

I'm wondering if the ui5-webcomponents-react library can easily be used with next.js and, if so, does an example project exist somewhere that I could reference? 

Thank you,
Quintin

**Comment by MarcusNotheis** [comment, marked as answer]
Hi @qscheitlin,

actually we're currently working on supporting Server Side Rendering Frameworks like Next.js. You can track the progress in #4091.

Current Status:
We already have some internal apps running with NextJS 13, but the developer experience is not yet as smooth as I would like to have it to be honest, so we'll need to iron out a couple of things first before adding it our documentation and promote it as general available.
The two main issues we're having currently is that you need to list all `@ui5/` packages in the `transpileModules` part in the `next.config.js` and that icon imports are somehow flaky and every now and then some icons are missing on the page.

I'm confident that we can smoothen this story in the next couple of weeks, we're planning to announce support for Next.js this quarter with an official example project.

Best regards,
Marcus

**Reply by qscheitlin** [reply]
Very cool to hear -- Thanks for the update, Marcus!
